README.md

```diff
@@ -9,10 +9,11 @@ The repository is structured as follows:
 - [Quantum machine learning and optimization with PennyLane](#pennylane)
 - [Amazon Braket features](#braket)
 - [Amazon Braket Hybrid Jobs](#jobs)
-- [Creating a Conda environment](#conda)
+- [Pulse Control](#pulse)
```

Comment:
This table of contents is good to have but the title (`The repository is structured as follows:`) doesn't clearly define what the purpose is.
I would argue changing it to 
`The examples structured as follows:`
does a better job of conveying this.